kickstarts/base.ks.erb

```diff
@@ -3,6 +3,7 @@
 logging --level=debug
 
 #version=DEVEL
+text
```

Comment:
I don't see any downside to running in text mode.  You can't even see any errors in graphical mode because you have to click something to display it and we usually just get a screenshot at the end.